shared/src/models/personnel.ts

```diff
@@ -32,8 +26,17 @@ export class Personnel {
     @IsUUID(4, uuidValidationOptions)
     public readonly vehicleId: UUID;
 
-    @IsLiteralUnion(personnelTypeAllowedValues)
-    public readonly personnelType: PersonnelType;
+    @IsString()
+    public readonly personnelType: string;
+
+    @IsUUID()
+    public readonly baseTemplateId: UUID;
```

Comment:
Same thing as with the material template: Name is redundant and validator is lacking args